### ISSUE TITLE
why did you use log-softmax in student-net, and softmax in teacher-net?

### ISSUE THREAD

**Issue by maliho0803** [NVlabs/DG-Net#18]
why did you use log-softmax in student-net, and softmax in teacher-net?

**Comment by RonakDedhiya** [comment]
I have the same question. What is the possible reason?
Is it is for soft labelling purpose ?

**Comment by layumi** [comment]
Hi @RonakDedhiya  
Thanks. You may check the KL-Divergence in Pytorch Document.
They provide the formulation, and you could  easily understand why I did it.
